intake/source/discovery.py

```diff
@@ -84,21 +83,31 @@ def scanned(self):
             self._scanned = _package_scan()
         return self._scanned
 
-    def enabled_plugins(self):
+    def disabled(self):
+        if self._disabled is None:
+            self._disabled = [k for k, v in self.conf.get("drivers", {}).items() if v is False]
+
+        return self._disabled
```

Comment:
Maybe it should be a set to prevent us from accidentally adding the same name twice?

Reply:
A test of this would be to disable a given name twice and then enable it once - is is still in the disabled list?

Reply:
This should definitely be a set